### PR TITLE
dev-python/pysvg: version bump, add python3 support

### PR DESCRIPTION
**Pull request by MeisterP**
Closes: https://bugs.gentoo.org/702518
Package-Manager: Portage-2.3.79, Repoman-2.3.16
Signed-off-by: Poncho <poncho@spahan.ch>

### CODE REVIEW

**Comment by MeisterP** [comment]
```diff
--- pysvg-0.2.2.ebuild	2019-10-14 11:48:04.450136868 +0200
+++ pysvg-0.2.2_p3.ebuild	2019-12-11 08:43:43.059020798 +0100
@@ -1,17 +1,20 @@
 # Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
-PYTHON_COMPAT=( python2_7 )
+EAPI=7
+PYTHON_COMPAT=( python3_{5,6,7,8} )
 
 inherit distutils-r1
 
+MY_PN="${PN}-py3"
+MY_PV="${PV/_p/.post}"
+
 DESCRIPTION="Python SVG document creation library"
-HOMEPAGE="https://codeboje.de/pysvg/"
-SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.zip"
+HOMEPAGE="https://github.com/alorence/pysvg-py3"
+SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_PN}-${MY_PV}.tar.gz"
 
 LICENSE="BSD"
 SLOT="0"
 KEYWORDS="~amd64"
 
-DEPEND="app-arch/unzip"
+S="${WORKDIR}/${MY_PN}-${MY_PV}"
```

File: dev-python/pysvg/pysvg-0.2.2_p3.ebuild [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{5,6,7,8} )
```
Comment (flappyports) [inline dev-python/pysvg/pysvg-0.2.2_p3.ebuild]: Please add a newline between EAPI and PYTHON_COMPAT

Comment (flappyports) [inline dev-python/pysvg/pysvg-0.2.2_p3.ebuild]: Please drop py3_5 as it will be going away in Gentoo

**Comment by flappyports** [comment]
@MeisterP, a few "nitpicks", but most importantly, your Sign-Off is missing a "full name." Please include your first/last name on the sign-off line.

Thank you so much for the PR. Will merge once fixed.

**Pull request closed without merging**